data/language/ko-KR.yml

```diff
@@ -1214,7 +1214,7 @@ strings:
   1184: "{POP16}{POP16}{POP16}{POP16}{POP16}{STRINGID}(을)를 {STRINGID}에 추가할 수 없습니다..."
   1185: "{POP16}{POP16}{POP16}{POP16}{POP16}{STRINGID}(을)를 지을 수 없습니다..."
   1186: "{COLOUR BLACK}새 차량을 선택하세요"
-  1187: "{COLOUR BLACK}{STRINGID}{STRINGID}"
+  1187: "{COLOUR BLACK}{STRINGID}"
```

Comment:
I wrote a script that automatically syncs the input tokens, this has been apparently been wrong.

Reply:
Interesting! The English string reads `1187: "{COLOUR BLACK}Select vehicle to add to {STRINGID}"`, so it seems something was omitted.

Reply:
It apparently never had text, the second string id is definitely too much, that would be reading out of bounds data.